### PR TITLE
feat(global): CI/CD 

### PR DESCRIPTION
**Pull request by rimiti**
## PURPOSE

This PR aims to add a CI/CD on the project.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    name: release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```
Comment (Simobain) [inline .github/workflows/release.yml:12]: What does checkout do ?

**Review by Simobain — APPROVED** [review]
One question (just being curious) but LGTM

File: .github/workflows/release.yml (hunk 1 above)
Comment (rimiti) [reply .github/workflows/release.yml:12]: It checkouts the repo.

**Review by Theo-Le-Donne — APPROVED** [review]
LGTM

**Pull request merged**